TransactionService/Transactions.py

```diff
@@ -170,12 +170,13 @@ def cancelBuy(userID):
 # Creates a sell request to be confirmed by the user
 def sell(userID, stockSymbol, amount):
     if cache.exists(userID + "_" + stockSymbol):
-        user = cache.get(userID)
+        user = json.loads(cache.get(userID))
         price = quote(userID, stockSymbol)
-        value = amount * price
-        if user["stock_amount"] >= amount:
-            cache.set(userID + "_SELL", {"user_id": userID, "stock_id": stockSymbol,
-                                         "amount": amount, "value": value, "time": datetime.datetime.now()})
+        amountOfStock = ceil(float(amount)/float(price))
```

Comment:
do we always want to take the ceiling here?

Reply:
In order to satisfy selling for that amount of money, we need to take the ceiling in order to find out how many whole shares of the stock we need to sell. Now that I think about it though, there is a bug there, but not related.